src/Doctrine/Orm/composer.json

```diff
@@ -27,7 +27,7 @@
         "api-platform/doctrine-common": "^4.2.9",
         "api-platform/metadata": "^4.2",
         "api-platform/state": "^4.2.4",
-        "doctrine/orm": "^2.17 || ^3.0"
+        "doctrine/orm": "^2.17 || ^3.0.1"
```

Comment:
It needs allowing ArrayParameterType for uuid binary in QueryBuilder https://github.com/doctrine/orm/pull/11287. 

So, Uuid binary filter does not work with Doctrine 2.x. Can I skip the tests if Doctrine 2.x is installed and throws an exception in the UuidBianryFilter ?